base/cvd/libbase/BUILD.bazel

```diff
@@ -34,6 +35,7 @@ cc_library(
         "mapped_file.cpp",
         "parsebool.cpp",
         "parsenetaddress.cpp",
+        "properties.cpp",
```

Comment:
could we use libbase from android too? If you do that first you won't need to fix this copy.